File: 프로그래머스/unrated/135808. 과일 장수/과일 장수.kt
class Solution {
    fun solution(k: Int, m: Int, score: IntArray): Int {
        var answer = 0
        val list = score.toMutableList().apply {
            sortDescending()
        }

        for(i in m-1 until list.size step m) {
            answer += m * list[i]
        }

        return answer
    }
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.swift
func solution(_ k:Int, _ m:Int, _ score:[Int]) -> Int {
    var answer = 0
    var sortedScore = score.sorted(by: >)
    
    for i in stride(from: m-1, to: sortedScore.count, by: m) {
        answer += sortedScore[i]
    }
    
    return answer * m
}